src/components/calendar/calendar.jsx

```diff
@@ -37,6 +37,10 @@ function getHeadCell(iterator) {
 }
 
 function isSameDate(date1, date2) {
+  if ((!date1 && date2) || (date1 && !date2)) {
+    return false;
+  }
```

Comment:
I think we've skipped the (perhaps contrived or unrealistic) case where _both_ are falsey. Is it just not needed?

Reply:
I really hope we're never calling this function in the case where the args are `null, null`, and protecting a function call where all params are `null` isn't really a pattern I want to start getting into. 😅

Reply:
In our case, one of the params is always present. We use this helper method to figure out whether the thing we're rendering (that represents a date) is equal to some state (another date).